interfaces/seccomp/backend.go

```diff
@@ -235,13 +235,37 @@ func (b *Backend) Remove(snapName string) error {
 	return nil
 }
 
+// Obtain the privilege dropping snippet
+func uidGidChownSnippet(name string) (string, error) {
+	tmp := strings.Replace(privDropAndChownSyscalls, "###USERNAME###", name, -1)
+	return strings.Replace(tmp, "###GROUP###", name, -1), nil
+}
+
 // deriveContent combines security snippets collected from all the interfaces
 // affecting a given snap into a content map applicable to EnsureDirState.
 func (b *Backend) deriveContent(spec *Specification, opts interfaces.ConfinementOptions, snapInfo *snap.Info) (content map[string]*osutil.FileState, err error) {
 	// Some base snaps and systems require the socketcall() in the default
 	// template
 	addSocketcall := requiresSocketcall(snapInfo.Base)
 
+	var uidGidChownSyscalls bytes.Buffer
+	if len(snapInfo.SystemUsernames) == 0 {
+		uidGidChownSyscalls.WriteString(barePrivDropSyscalls)
+	} else {
+		firstRun := true
```

Comment:
This is never set to false (and not tested hence we did not notice). I also wonder if we really need it, here is a diff with/without:
```
-- /tmp/no-rune	2019-08-15 10:37:09.848342485 +0200
+++ /tmp/with-rune	2019-08-15 10:39:46.599749824 +0200
@@ -627,6 +627,7 @@
 lchown - -1 g:testid
 lchown32 - -1 g:testid
 
+
 # allow setgid to testid2
 setgid g:testid2
 setgid32 g:testid2
```
given that we already have a \n there I don't think we really need it.